src/bin/cargo/commands/owner.rs

```diff
@@ -6,46 +6,137 @@ use cargo_credential::Secret;
 pub fn cli() -> Command {
     subcommand("owner")
         .about("Manage the owners of a crate on the registry")
-        .arg(Arg::new("crate").value_name("CRATE").action(ArgAction::Set))
+        .arg_quiet()
+        .arg(Arg::new("crate").hide(true))
```

Comment:
Something we've not talked about is the stabilization plan.

Options
- Insta-stable: requires us being more confident, will have an FCP in this issue for the team to agree
- Put the subcommands behind `-Zunstable-options` and stabilize later
  - We'd want to keep the focus on the existing flags until the stabilization PR

Another benefit to stabilizing later is we can re-evaluate this command to see if there are any "breaking" changes we want to make.  For example, would we want `add` and `remove` to be dry-run by default?

Reply:
I am learning the operating rules of the community and I can cooperate with your decisions.